### DISCUSSION TITLE
Backup and restore is giving an issue

### DISCUSSION THREAD

**Discussion by ciersel** [teslamate-org/teslamate#2530] [Q&A]
Hi, 

Hope someone can help me out.
Have a raspberry running with Teslamate. Is giving some strange issues.
Want to move to another raspberry. Followed the backup and restore document.
Made a backup ( also took the docker-compose.yml ) Installed the new raspberry with the same .yml file
When I try to look at my date in grafana there is no data. The layout in Grafana is there.

<img width="1483" alt="image" src="https://user-images.githubusercontent.com/101416726/157904544-9aeb6709-ddba-4c3f-a33e-3dcc32de7809.png">

Anybody an idea what I'm doing wrong ?

Thanks,
Charly

**Comment by cwanja** [comment]
Did you restore the file? Do any other dashboards show anything? 

You can always try and drop the current database and restore it, following the instructions here https://docs.teslamate.org/docs/maintenance/backup_restore/#restore

**Reply by ciersel** [reply]
Thanks for your reply !
Did the restore and all the other dashboards were empty too.

Just did made a new backup and restored it on the new pi and now the data is showing up. No idea what I did wrong. But the data is showing up again. Now an upgrade to the latest version and implement the fix for libseccomp2

Do you happen to know if you can run 2 instances at the same time. Does Tesla " accept" this

**Reply by cwanja** [reply]
>Do you happen to know if you can run 2 instances at the same time. Does Tesla " accept" this

You can, but you could run into API throttle issues. I would not advise this, but you can. It is no different than you running TeslaMate and TeslaFi or TeslaScope or even just the first-party Tesla app.

**Reply by woyteck1** [reply]
I ran two of these for a test. The Teslamate will start showing Healthcheck Failed icon randomly on either of instances.

**Reply by cwanja** [reply]
Just remember that is TeslaMate reporting an error with the API response. Check the Tesla mobile app, you will probably see an error at the same time that TeslaMate reports a health check failure.

**Comment by woyteck1** [comment]
When I migrated to other machine, after docker installation, I had to login to postgres and drop tokens table. THen I imported my data I think. Can't confirm, because containers do not remember postgres history if you stop them :/
